components/org.wso2.carbon.identity.admin.session.advisory.banner.control/src/main/java/org/wso2/carbon/identity/admin/session/advisory/banner/control/handler/AdminSessionAdvisoryBannerControlHandler.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2023, WSO2 Inc. (http://www.wso2.com).
```

Comment:
Should be WSO2 LLC

Reply:
resolved with 1bc0c4ef78c1f9502e402d0c5fd2494d51103f04